### DISCUSSION TITLE
Hide abandoned packages from the Grid view

### DISCUSSION THREAD

**Discussion by adamn** [djangopackages/djangopackages#651] [Grids]
I've been using Django for many years and was recently showing it to a new developer.  When showing how great it is I realized that there is a bunch of cruft on the site now.  Some packages haven't been updated in years and should not be shown to users looking for a solution to their problem.

I wonder if it makes sense to have an `in_grid` boolean for the `Package` model which defines whether the package shows up in the grid.

An inactive package should still have it's individual package url but I think the grids would be cleaner and the site itself would be more delightful.

An easy heuristic to do the initial cull is via Python3 support with maybe 1 or 2 manual exceptions.  Any package that doesn't yet have Python3 support is probably dead and likely won't work with Django 1.10, let alone 1.11 (and certainly not Django 2 of course).

This also means that the Python 3 support row can be removed from the grids as well - further cleaning things up.

**Comment by fabiocaccamo** [comment]
Any news about this request?

There is no reason to show packages which are not updated since more than 2 years and doesn't provide python 3 support. They could be hidden, or moved at the end of the list.

The last update date should have more weight in the ordering, I prefer to try a new package with 10 stars vs an abandoned one with 1k stars.

**Comment by RamezIssac** [comment]
I think an approach for this can the ordering of the packages ..
The packages are ordered by the github stars (as far as i see).

Maybe it should ordered be a function of the stars and the last update date

A proposed formula can be : Actual stars -10% for each 3 months of inactivity - 30% if not supporting python 3.
